src/aat/resources/features/F-049/S-232.td.json

```diff
@@ -0,0 +1,23 @@
+{
+  "_guid_": "S-232",
+  "_extends_": "F-049_Test_Data_Base",
+  "title": "must return 403 when request provide unauthorized access",
+  "specs": [
+    "a detailed profile in CCD",
+    "Start case creation as Case worker",
+    "CCD Data Store",
+    "provide unauthorized access",
+    "code is HTTP-403"
+  ],
+  "request": {
```

Comment:
where is the unauthorized access in this request?

Reply:
as we are not allowed to change the user name and password hence changing the JID which is not accessible to the user, is it make sense please let me know if some other option we have

Reply:
as we can't change the user name and password hence changing the **unauthorized jurisdiction id **

Reply:
scenario title and content are inconsistent, then.